types/jqgrid/jqgrid-tests.ts

```diff
@@ -1,8 +1,10 @@
 // Type definitions for jQuery jqgrid Plugin 1.3
```

Comment:
A test file shouldn't even have a header, please remove it.

Reply:
OK